storybook/config.js

```diff
@@ -0,0 +1,18 @@
+import { configure } from '@kadira/storybook';
+import React from 'react';
+import { storiesOf, action } from '@kadira/storybook';
+
+import './storybook.css'
+// for now just simply $ rake asset:precompile && mv public/assets/application-ebf... storybook/application.css
```

Comment:
That's just a quick and dirty way to give the components the same style as in the app.  Unfortunately I have no clue what's the best way to handle component styling and I hadn't the chance to try radium etc yet. But it would be definitely nice if components would bring their own styles.